Datadog.Trace.Build.slnf

```diff
@@ -0,0 +1,8 @@
+{
+  "solution": {
```

Comment:
I don't think this file is necessary? 🤔 If you just want to open the build csproj... just open the build.csproj 😄